tasks/debian.yml

```diff
@@ -0,0 +1,40 @@
+- name: "Install exim (exim4) the mail transfer agent"
+  apt: >
+    name=exim4-daemon-heavy
+    state=present
+  tags:
+    - exim
+
+- name: "Make sure exim (exim4) is running"
+  service: >
+    name=exim4
+    state=started
+    enabled=yes
+  tags:
+    - exim
+
+
+- name: "Configure the package"
+  shell: |
+    executable=/bin/bash
+    dpkg-reconfigure -f noninteractive exim4-config &&\
+    update-exim4.conf
+  tags:
+    - exim
+
+- name: "Copy the template configuration"
+  template: >
+    src=update-exim4.conf
+    dest=/etc/exim4/update-exim4.conf
+    owner=root
+    group=root
+    mode=0644
+  tags:
+    - exim
+
+- name: "Restart the exim (exim4) service"
+  service: >
+    name=exim4
+    state=restarted
```

Comment:
Should be `enable=yes` and `state=running` to be sure it will restart with the server, and to be sure it's currently running.
If you use `state=restarted` it will restart every time you provision.